src/screens/TokenDetail.js

```diff
@@ -187,10 +215,36 @@ class TokenDetail extends React.Component {
       )
     }
 
+    const renderNftPreview = () => {
+      if (this.state.nft) {
+
+        let media;
+
+        if (this.state.nft.type === 'image') {
+          media = <img src={this.state.nft.file} width="100%" height="100%" />;
+        } else {
+          media = (
+            <video controls>
```

Comment:
What do we have in the three dots in the video player? I think we should at least remove the download option. We could leave volume, full screen but (even though we know it can be downloaded anyway) we should try to hide this download button. This below should work:

```
controlsList="nodownload"
```

Reply:
Why should we remove the download option? I feel that people are free to download the asset behind an NFT.

Reply:
I agree that people are free but (i) I think the player is small and have many buttons, thus removing the 3 dots icon would be better imo for UI and (ii) the idea of the NFT detail in the explorer is to show, not download (if the idea was to have a download option the design would have it).

Reply:
I agree. We can move forward without a download button.